src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const totalCost = days * dailyRate;
+  const smallDiscount = 20;
+  const discount = 50;
+  let cost = 0;
+
+  if (days < 3) {
+    return totalCost;
+  }
```

Comment:
While the code correctly returns the `totalCost` for rentals less than 3 days, consider using a constant to represent the minimum days required for a discount (e.g., `const MIN_DAYS_FOR_DISCOUNT = 3`). This will make the code more readable and maintainable.